extensions/amp-story-dev-tools/0.1/amp-story-dev-tools-tab-logs.css

```diff
@@ -115,3 +115,32 @@ amp-story-dev-tools-tab-logs {
+  width: 260px !important;
+  height: 300px !important;
+  margin-bottom: 20px !important;
+}
+
+.i-amphtml-story-dev-tools-logs-success {
+  display: flex !important;
+  align-items: center !important;
+  justify-content: center !important;
+  flex-direction: column;
+  height: 100% !important;
+  white-space: pre !important;
+  text-align: center !important;
```

Comment:
nit: this could be on the success message node.
It doesn't appear to have any side effects but there are other text children. It would be more readable to have this on the specific node that is intended to have it.